pretty/src/pretty/pretty_doc.rs

```diff
@@ -34,32 +34,39 @@ pub trait PrettyDocument: Sized + Clone {
     /// result.**
     fn bounds(&self) -> Bounds;
 
-    /// Render the document onto the screen. This method behaves as if it did
+    /// Render the document onto a `PrettyPane`. This method behaves as if it did
     /// the following:
     ///
     /// 1. Pretty-print the entire document with width `width`.
-    /// 2. Transcribe the portion of the document under the screen onto the
-    /// screen. (Remember that the screen is located at `screen.region()`.)
+    /// 2. Position the document under the `PrettyPane`, aligning `doc_pos` with the `PrettyPane`'s upper left corner.
+    /// 3. Render the portion of the document under the `PrettyPane` onto the `PrettyPane`.
     ///
     /// However, this method is more efficient than that, and does an amount of
-    /// work that is (more or less) proportional to the size of the screen,
+    /// work that is (more or less) proportional to the size of the `PrettyPane`,
     /// regardless of the size of the document.
```

Comment:
Say: what's `doc_pos` about? `width`?

Reply:
It says: "...aligning `doc_pos` with the `Pane`'s upper left corner". Is there a different explanation of `doc_pos` you're looking for?